pandas/io/json/_normalize.py

```diff
@@ -229,14 +230,23 @@ def _json_normalize(
     Returns normalized data with columns prefixed with the given string.
     """
 
-    def _pull_field(js, spec):
-        result = js
+    def _pull_field(js: Dict[str, Any], spec: Union[List, str]) -> Iterable:
+        result = js  # type: ignore
```

Comment:
Why do you need the `type: ignore` here?

Reply:
See: https://github.com/pandas-dev/pandas/pull/30145#discussion_r358483282 and the thread above